index.html

```diff
@@ -0,0 +1,1591 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="main.css" />
+    <title>TNW</title>
+    <script
+      src="https://kit.fontawesome.com/63c1245c39.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <!-- Navigator -->
+    <nav class="nav nav-m-b">
```

Comment:
The top section(the one with the dark background) of the navbar is missing in your clone. Please add that.

| Source Page	| Clone Page |
|:---:|:---:|
|![Annotation 2020-07-10 024642](https://user-images.githubusercontent.com/57726348/87091671-b7370100-c257-11ea-9da6-abf24c56e82e.jpg)|![image](https://user-images.githubusercontent.com/57726348/87091643-a9817b80-c257-11ea-8d7d-9a3ddaf6a89a.png)|